lib/internal/process/promises.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 
 const { safeToString } = process.binding('util');
+const abortRegex = /^--abort[_-]on[_-]unhandled[_-]rejection$/;
```

Comment:
Do we need to support the `_` variant? We do for `--abort-on-uncaught-exception`, but that's because it's a V8 option.

Reply:
It’s here for parity with `--abort_on_uncaught_exception` but yes, we can choose to not support it if we like.

Reply:
It doesn't cost much to keep parity; I would suggest keeping it.

Reply:
I'm +0 either way but if we do support it then we should probably have a test variant for it.